instrumentation/opentelemetry-instrumentation-sqlite3/src/opentelemetry/instrumentation/sqlite3/__init__.py

```diff
@@ -104,9 +119,14 @@ def instrument_connection(
                 the current globally configured one is used.
 
         Returns:
-            An instrumented connection.
-        """
+            An instrumented SQLite connection that supports
+            telemetry for tracing database operations.
 
+        Notes:
+            - Instrumentation must be explicitly applied to the connection object
+              for tracing to work. This is not done automatically by simply calling
+              `SQLite3Instrumentor().instrument()`.
+        """
```

Comment:
It should actually work if user does `SQLite3Instrumentor().instrument()` then do e.g. `cnx = sqlite3.connect(':memory:')` and `cursor = cnx.cursor()`. Were there issues when you did the tests update?